test/IRGen/class_update_callback_without_fixed_layout.sil

```diff
@@ -6,7 +6,7 @@
 
 // REQUIRES: objc_interop
 // UNSUPPORTED: OS=iosmac
-// UNSUPPORTED: CPU=arm64e
+// UNSUPPORTED: CPU=arm64 || CPU=arm64e
```

Comment:
I thought we had a better way to conditionalize tests on targets that don't have a %target-pre-stable-abi-triple?

Reply:
We don't, but, I like your idea, I'll see if I can whip up something.